Release.toml

```diff
@@ -12,4 +12,11 @@ version = "1.0.4"
 "(1.0.1, 1.0.2)" = ["migrate_v1.0.2_add-enable-spot-instance-draining.lz4"]
 "(1.0.2, 1.0.3)" = ["migrate_v1.0.3_add-sysctl.lz4"]
 "(1.0.3, 1.0.4)" = []
-"(1.0.4, 1.0.5)" = ["migrate_v1.0.5_add-lockdown.lz4", "migrate_v1.0.5_sysctl-subcommand.lz4", "migrate_v1.0.5_add-user-data.lz4"]
+"(1.0.4, 1.0.5)" = [
+    "migrate_v1.0.5_add-lockdown.lz4",
+    "migrate_v1.0.5_sysctl-subcommand.lz4",
+    "migrate_v1.0.5_add-user-data.lz4",
+    "migrate_v1.0.5_add-network-settings.lz4",
+    "migrate_v1.0.5_replace-service-settings.lz4",
+    "migrate_v1.0.5_add-services.lz4"
```

Comment:
I know it's a nitpick, but these names are effectively in a single namespace forever, and this is fairly generic without any reference to which services or the reason for updating/adding services; can we make them a little more specific?  Like "add-proxy-restart" and "add-proxy-services" or something?  They're still fundamentally tied to the proxy feature, given that they add proxy-env, not just restarts.